validator/validator-main.protoascii

```diff
@@ -3145,6 +3145,19 @@ tags: {
     }
   }
 }
```

Comment:
@shawnbuso You'll also want the `cdata` section as we don't allow html comments within `script` tags.

@Gregable as fyi for validation review.